web_client/models/AssetstoreModel.js

```diff
@@ -0,0 +1,37 @@
+import AssetstoreModel from 'girder/models/AssetstoreModel';
+import { restRequest } from 'girder/rest';
+
+/**
+ * Extends the core assetstore model to add Database-specific functionality.
+ */
+AssetstoreModel.prototype.databaseImport = function (params) {
+    restRequest({
+        path: 'database_assetstore/' + this.get('_id') + '/import',
+        type: 'PUT',
+        data: params,
+        error: null
+    }).done(() => {
+        this.trigger('g:imported');
+    }).fail((err) => {
+        this.trigger('g:error', err);
+    });
+
+    return this;
```

Comment:
Since we're making backward incompatible changes, returning the promise from the rest request would probably be better.